src/lib/dlt_user.c

```diff
@@ -2799,7 +2799,10 @@ void dlt_user_trace_network_segmented_thread(void *unused)
         if (read < 0) {
             if (errno != EINTR) {
                 dlt_vlog(LOG_WARNING, "NWTSegmented: Error while reading queue: %s\n", strerror(errno));
-                usleep(DLT_USER_MQ_ERROR_RETRY_INTERVAL);
+                struct timespec req;
```

Comment:
Please put it before dlt_vlog(). Declaration should go to the beginning.

Reply:
done